### PR TITLE
Formatted operators-detail components

### PR DESCRIPTION
**Pull request by blayhem**
Eslint config and components formatting (on operators-detail only, for now).

### CODE REVIEW

File: components/operators-detail/documentation/documents-by-operator.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ import { HELPERS_DOC } from 'utils/documentation';
 // Components
 import DocCard from 'components/ui/doc-card';
 import DocCardUpload from 'components/ui/doc-card-upload';
-import DocumentsByFMU from './documents-by-fmu';
+import DocumentsByFMU from './documents-by-fmu.js';
```
Comment (mbarrenechea) [inline components/operators-detail/documentation/documents-by-operator.js:18]: you don't need to add .js right?

**Review by mbarrenechea — APPROVED** [review]
There is only one little thing, you add a .js when you were importing one file

File: components/operators-detail/documentation/documents-by-operator.js (hunk 1 above)
Comment (blayhem) [reply components/operators-detail/documentation/documents-by-operator.js:18]: `eslint` asks for extension when the import is relative. I guess I could disable that rule, too. Check https://github.com/Vizzuality/otp-portal/blob/feature/operators-detail-format/.eslintrc#L49 and let's agree on a common rule/solution?

Comment (mbarrenechea) [reply components/operators-detail/documentation/documents-by-operator.js:18]: Yes, disable it

Comment (blayhem) [reply components/operators-detail/documentation/documents-by-operator.js]: Done, you can merge if you want 👍

**Pull request merged**